src/components/navigation/CSidebar.vue

```diff
@@ -191,6 +191,18 @@ export default {
     },
   },
 
+  created () {
```

Comment:
If this will be here it needs to be removed from compose. Pick one

Reply:
I removed it in compose ( and that event this.$root.$on('close-sidebar') and moved in corteza-vue

Reply:
Lets make sure this is in tomorrows dev-1 release